test/utilities/miscellaneous.ts

```diff
@@ -26,7 +26,7 @@ export function log(message: string): void {
 }
 
 export function debug(message: string): void {
-  if (EnvironmentSettings.getInstance().logLevel in ['debug', 'trace']) {
+  if (EnvironmentSettings.getInstance().logLevel in ['debug', 'trace', 'warn']) {
```

Comment:
To be in line with accepted understanding of log level, this should include `info` as well.